client/knode/validator/validator.go

```diff
@@ -160,7 +160,14 @@ func (val *validator) run() {
 
 	log.Info("Starting the consensus state machine")
 	for state, numTransitions := val.notLoggedInState, 0; state != nil; numTransitions++ {
```

Comment:
Is this valid? before it only returned one param notLoggedInState, but now it returns 2. Will it work???

Reply:
hm, strange apparently does work, but your right.

Maybe has unexpected behaviour, let me do more testing